crates/host/src/secrets.rs

```diff
@@ -138,25 +138,27 @@ impl Manager {
             .then(|secret_name| async move {
```

Comment:
I _think_ we need to massage the secret_name too.
https://github.com/wasmCloud/wasmCloud/blob/main/crates/host/src/wasmbus/mod.rs#L3330
It's likely getting here as `SECRET_rust_hello_world-example`

Reply:
Oh, I see the issue here. When the app is created by wadm we automatically prepend the application name to the name of the secret to prevent collisions between manifests.

That being said, the component shouldn't be responsible for knowing what app it's a part of, nor should the host be responsible for stripping an app name. I'll need to check to see if the host strips this out anywhere or how exactly it works.

Reply:
@lxfontes fixed this in https://github.com/wasmCloud/wasmCloud/pull/2564/commits/eb76e5204d3d494eefd3146f26bda6e0d7ca71de